v03_pipeline/lib/tasks/write_remapped_and_subsetted_callset.py

```diff
@@ -127,9 +127,9 @@ def create_table(self) -> hl.MatrixTable:
 
         loadable_families = (
             families
-            - families_failed_missing_samples
-            - families_failed_relatedness_check
-            - families_failed_sex_check
+            - families_failed_missing_samples.keys()
+            - families_failed_relatedness_check.keys()
+            - families_failed_sex_check.keys()
```

Comment:
does it happen often that all families fail the checks? is it worth changing the error message on line 135 to include info about which families failed which checks?